hive-metastore/src/main/java/org/apache/iceberg/hive/HiveCatalog.java

```diff
@@ -61,12 +62,16 @@
 import org.slf4j.LoggerFactory;
 
 public class HiveCatalog extends BaseMetastoreCatalog implements SupportsNamespaces, Configurable {
+  public static final String LIST_ALL_TABLES = "hive.catalog.list-all-tables";
```

Comment:
This is too much nesting for a catalog config property. We try to keep them as short as possible because these are embedded in Spark or other configs. Luckily, you don't need `hive` or `catalog`. Both of those are clear from context. This is catalog config passed to a command like `CREATE CATALOG` so including `catalog` would be redundant. And `hive` is unnecessary because the config is for a Hive catalog. We don't prefix config properties with the catalog they are intended for.

You can change this to just `list-all-tables` and it will be good to go.

Reply:
ok, will rename to list-all-tables.